index.html

```diff
@@ -50,6 +58,203 @@ <h1>Hey there.
+                        <p class="cardsupporting">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. 
+                            has been the industry's standard
+                        </p>
+                        <ul class="tags catags">
+                            <li class="taging cataging"><p class="tag catag">html</p></li>
+                            <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                            <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                        </ul>
+                    </section>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div> 
+
+                <div class="cards">
+                    <section class="rightblock">
+                        <h1 class="cardtitle">Profesional Art Printing Data</h1>
+                        <p class="cardsupporting">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. 
+                            has been the industry's standard
+                        </p>
+                        <ul class="tags catags">
+                            <li class="taging cataging"><p class="tag catag">html</p></li>
+                            <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                            <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                        </ul>
+                    </section>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div> 
+
+                <div class="cards">
+                    <section class="rightblock">
+                        <h1 class="cardtitle">Profesional Art Printing Data</h1>
+                        <p class="cardsupporting">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required. 
+                            has been the industry's standard
+                        </p>
+                        <ul class="tags catags">
+                            <li class="taging cataging"><p class="tag catag">html</p></li>
+                            <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                            <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                        </ul>
+                    </section>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div> 
+
+            </div>          
+        </section>
+
+
+        <section class="aboutme">
```

Comment:
- [ ] I noticed here the font family is not the same as the Figma template kindly update it to be the same. 👌🏻